From: Matthias Fehring <buschmann23@opensuse.org>
Date: 2019-09-17 23:42:00 +0200
Subject: README.md has unicode in it
Upstream: merged (https://github.com/redis/hiredis-py/pull/89)

When reading the description in setup.py the wrong encoding is
selected for the text files.

---
 setup.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index bc8a95c..7ec9a5b 100755
--- a/setup.py
+++ b/setup.py
@@ -4,7 +4,7 @@
   from setuptools import setup, Extension
 except ImportError:
   from distutils.core import setup, Extension
-import sys, imp, os, glob
+import sys, imp, os, glob, io

 def version():
   module = imp.load_source("hiredis.version", "hiredis/version.py")
@@ -19,7 +19,7 @@ def version():
   name="hiredis",
   version=version(),
   description="Python wrapper for hiredis",
-  long_description=open('README.md', 'r').read(),
+  long_description=io.open('README.md', 'rt', encoding='utf-8').read(),
   long_description_content_type='text/markdown',
   url="https://github.com/redis/hiredis-py",
   author="Jan-Erik Rediger, Pieter Noordhuis",
